Source/drlg_l1.cpp

```diff
@@ -679,34 +775,15 @@ bool PlaceMiniSet(const BYTE *miniset, bool setview)
 			continue;
 		}
 
-		int ii = 2;
-
-		bool success = true;
-		for (int yy = 0; yy < sh && success; yy++) {
-			for (int xx = 0; xx < sw && success; xx++) {
-				if (miniset[ii] != 0 && dungeon[xx + sx][sy + yy] != miniset[ii])
-					success = false;
-				if (Protected[xx + sx][sy + yy])
-					success = false;
-				ii++;
-			}
-		}
-		if (success)
+		if (SetPiecesRoom.Contains({ sx, sy }))
+			continue;

```

Comment:
I'd rather add braces here than remove them from the other instances 😂

Reply:
Normally I would agree, but for C++/DevilutionX I'm still wavering :)

Reply:
Braces guard against indendation errors but in DevilutionX they can't happen thanks to clang-format, so it's a purely stylistic preference at this point (I prefer less visual noise over consistent braces)